base/system/autochk/autochk.c

```diff
@@ -316,7 +316,7 @@ ChkdskCallback(
 
     case PROGRESS:
         Percent = (PDWORD) Argument;
-        PrintString("%d percent completed.\r", *Percent);
+        PrintString("%d percent completed\r", *Percent);
```

Comment:
Not sure I understand this change.

Reply:
Whoops, made a punctuation improvement then removed that, heh